kruize-clowdapp.yaml

```diff
@@ -0,0 +1,130 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: kruize
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: kruize
+  spec:
+    envName: ${ENV_NAME}
+    deployments:
+    - name: recommendations
+      replicas: ${{KRUIZE_REPLICA_COUNT}}
+      webServices:
+        private:
+          enabled: true
+      podSpec:
+        image: ${KRUIZE_IMAGE}:${KRUIZE_IMAGE_TAG}
+        command: ["sh"]
+        args: ["-c", "export DB_CONFIG_FILE=${ACG_CONFIG} && bash target/bin/Autotune"]
+        machinePool: ${MACHINE_POOL_OPTION}
+        resources:
+          requests:
+            cpu: ${KRUIZE_CPU_REQUEST}
+            memory: ${KRUIZE_MEMORY_REQUEST}
+          limits:
+            cpu: ${KRUIZE_CPU_LIMIT}
+            memory: ${KRUIZE_MEMORY_LIMIT}
+        env:
+          - name: AUTOTUNE_SERVER_PORT
+            value: ${KRUIZE_PORT}
+          - name: AUTH_TOKEN
+            value: ""
+          - name: LOGGING_LEVEL
+            value: "info"
+          - name: ROOT_LOGGING_LEVEL
+            value: "info"
+          - name: dbdriver
+            value: "jdbc:postgresql://"
+          - name: database_name
+            value: ${KRUIZE_DB_NAME}
+          - name: clustertype
+            value: "kubernetes"
+          - name: k8stype
+            value: "openshift"
+          - name: authtype
+            value: "openshift"
+          - name: monitoringagent
+            value: "prometheus"
+          - name: monitoringservice
+            value: "prometheus-k8s"
+          - name: monitoringendpoint
+            value: "prometheus-k8s"
+          - name: savetodb
+            value: "true"
+          - name: hibernate_dialect
+            value: "org.hibernate.dialect.PostgreSQLDialect"
+          - name: hibernate_driver
+            value: "org.postgresql.Driver"
+          - name: hibernate_c3p0minsize
+            value: "2"
+          - name: hibernate_c3p0maxsize
+            value: "5"
+          - name: hibernate_c3p0timeout
+            value: "300"
+          - name: hibernate_c3p0maxstatements
+            value: "50"
+          - name: hibernate_hbm2ddlauto
+            value: "update"
+          - name: hibernate_showsql
+            value: "false"
+          - name: hibernate_timezone
+            value: "UTC"
+          - name: SSL_CERT_DIR
+            value: ${SSL_CERT_DIR}
+    database:
+      name: kruize
+      version: 13
+    testing:
+      iqePlugin: ros-ocp
```

Comment:
I don't think we have any iqe tests specific to kruize so this should be removed.